warehouse/query-core/src/main/java/datawave/query/function/DocumentProjection.java

```diff
@@ -38,40 +38,43 @@ public class DocumentProjection implements DocumentPermutation {
     private boolean trackSizes = true;
 
     public DocumentProjection() {
-        this(false, false);
+        this.includeGroupingContext = false;
```

Comment:
Why not leave this as : this(false, false)

Reply:
I get a compilation failure when I do it that way.

Reply:
You can change this constructor to
```java
public DocumentProjection() {
    this(false, false);
}
```
without getting a compilation error

Reply:
This change wasn't made?

Reply:
Change has been made.